ci/buildwheel.sh

```
@@ -0,0 +1,64 @@
+  twine upload \
+    --repository local \
+    dist/* \
+    --config-file /root/.pypirc
+}
+
+function main() {
+  setup_root
+  setup_app
+  build_wheel
+  # additional safegard against uploading
+  if [ x"${BUILD_TYPE}" == x"release" ]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, == in place of = is undefined.

(SC3014)